src/util/SnowflakeUtil.js

```diff
@@ -16,7 +16,7 @@ class SnowflakeUtil extends null {
    * ```
    * 64                                          22     17     12          0
    *  000000111011000111100001101001000101000000  00001  00000  000000000000
-   *       number of ms since Discord epoch       worker  pid    increment
+   *       number of milliseconds since Discord epoch       worker  pid    increment
```

Comment:
This may not be 100% right but the text needs to align with the bits
```suggestion
   * number of milliseconds since Discord epoch       worker  pid    increment
```